packages/cache/src/utils/hasResetKeysChanged.ts

```diff
@@ -0,0 +1,2 @@
+export const hasResetKeysChanged = (a: unknown[] = [], b: unknown[] = []) =>
```

Comment:
```suggestion
```

Reply:
I modified it https://github.com/toss/suspensive/pull/1083/commits/ceb14e435a346cfb2f3a4ea546dc60bb7444a34a